src/main/java/uk/org/openbanking/datamodel/fund/OBFundsConfirmationConsent1Data.java

```diff
@@ -32,95 +31,99 @@
 /**
  * OBFundsConfirmationConsent1Data
  */
-@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-05-20T10:02:34.555654+01:00[Europe/London]")
-
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-11-18T09:57:44.731259Z[Europe/London]")
 public class OBFundsConfirmationConsent1Data {
-  @JsonProperty("ExpirationDateTime")
-  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
-  private DateTime expirationDateTime;
-
-  @JsonProperty("DebtorAccount")
-  private OBFundsConfirmationConsent1DataDebtorAccount debtorAccount;
-
-  public OBFundsConfirmationConsent1Data expirationDateTime(DateTime expirationDateTime) {
-    this.expirationDateTime = expirationDateTime;
-    return this;
-  }
-
-  /**
-   * Specified date and time the funds confirmation authorisation will expire.  If this is not populated, the authorisation will be open ended.All dates in the JSON payloads are represented in ISO 8601 date-time format.  All date-time fields in responses must include the timezone. An example is below: 2017-04-05T10:43:07+00:00
-   *
-   * @return expirationDateTime
-   */
-  @ApiModelProperty(value = "Specified date and time the funds confirmation authorisation will expire.  If this is not populated, the authorisation will be open ended.All dates in the JSON payloads are represented in ISO 8601 date-time format.  All date-time fields in responses must include the timezone. An example is below: 2017-04-05T10:43:07+00:00")
-  @Valid
-  public DateTime getExpirationDateTime() {
-    return expirationDateTime;
-  }
-
-  public void setExpirationDateTime(DateTime expirationDateTime) {
-    this.expirationDateTime = expirationDateTime;
-  }
-
-  public OBFundsConfirmationConsent1Data debtorAccount(OBFundsConfirmationConsent1DataDebtorAccount debtorAccount) {
-    this.debtorAccount = debtorAccount;
-    return this;
-  }
-
-  /**
-   * Get debtorAccount
-   *
-   * @return debtorAccount
-   */
-  @ApiModelProperty(required = true, value = "")
-  @NotNull
-  @Valid
-  public OBFundsConfirmationConsent1DataDebtorAccount getDebtorAccount() {
-    return debtorAccount;
-  }
-
-  public void setDebtorAccount(OBFundsConfirmationConsent1DataDebtorAccount debtorAccount) {
-    this.debtorAccount = debtorAccount;
-  }
-
-  @Override
-  public boolean equals(Object o) {
-    if (this == o) {
-      return true;
+    @JsonProperty("ExpirationDateTime")
+    @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME)
```

Comment:
Wouldn't better import the object instead of use the package path?

Reply:
That's how it was generated, so I didn't want to mess with it or next time we generate it there would be a difference to investigate...